images/driver-base/Dockerfile

```diff
@@ -12,15 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM photon:2.0
+FROM photon:3.0
 LABEL "maintainer" "Travis Rhoden <trhoden@vmware.com"
```

Comment:
do you want to set yourself as the maintainer now?

Reply:
Oh, and if you do change it, should probably fix my typo of the missing `>` at the end of the email address. 😆

Reply:
I agree with @codenrhoden to change the maintainer.

Reply:
done. also fixed dockerfile syntax. It should be ```LABEL <key>=<value>```.